mne/io/fieldtrip.py

```diff
@@ -0,0 +1,345 @@
+# -*- coding: UTF-8 -*-
+# Copyright (c) 2018, Thomas Hartmann & Dirk Gütlin
+# Authors: Thomas Hartmann <thomas.hartmann@th-ht.de>
+#          Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+# License: BSD (3-clause)
+
+import mne
+import numpy as np
+
+def _check_pymatreader():
+    """Helper to check if pymatreader and h5py are installed"""
+    try:
+        from ..externals import pymatreader
+    except ImportError:
+        raise ImportError('The h5py module is required to use the FieldTrip '
+                          'import functions.')
+    return pymatreader
+
+def read_raw_fieldtrip(ft_structure_path, data_name='data'):
+    """Load continuous (i.e. raw) data from a FieldTrip preprocessing structure.
+
+    This function expects to find single trial raw data (FT_DATATYPE_RAW) in
+    the structure data_name is pointing at.
+
+    Parameters
+    ----------
+    ft_structure_path: str
+        Path and filename of the .mat file containing the data.
+    data_name: str
+        Name of heading dict/ variable name under which the data was originally
+        saved in MATLAB.
+
+    Returns
+    -------
+    mne.io.RawArray
+        A MNE RawArray structure consisting of the raw array and measurement
+        info
+
+    """
+    pymatreader = _check_pymatreader()
+
+    ft_struct = pymatreader.read_mat(ft_structure_path,
+                                     ignore_fields=['previous'],
+                                     variable_names=[data_name])
+
+    # load data and set ft_struct to the heading dictionary
+    ft_struct = ft_struct[data_name]
+
+    data = np.array(ft_struct['trial'])  # create the main data array
+    info = _create_info(ft_struct)  # create info structure
+
+    custom_raw = mne.io.RawArray(data, info)  # create an MNE RawArray
+    return custom_raw
+
+
+def read_epochs_fieldtrip(ft_structure_path, data_name='data',
+                         trialinfo_map=None,
+                   omit_trialinfo_index=True,
+                   omit_non_unique_trialinfo_index=True):
```

Comment:
More PEP8 problems -- it's probably worth getting the style checking running locally (and I recommend using an editor with automated PEP8 checking enabled)